kano_settings/set_audio.py

```diff
@@ -82,6 +111,12 @@ def current_setting(self):
         analogue_supported = is_analogue_audio_supported()
         hdmi_selected = is_HDMI()
 
+        if not hdmi_supported:
+            self.disabled_hdmi_label.get_style_context().add_class('normal_label')
```

Comment:
This line effectively shows the message and it does so by setting the styling of the text from a Gtk default of `white`...